### PR TITLE
Fix latest COVs, few indents and remove some dead code here and there and remove few globals

### PR DESCRIPTION
**Pull request by trufae**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by XVilka** [comment]
It has broken many tests:
```
[XX] C:\projects\radare2\test\db\cmd\cmd_agC agC function names are resovled properly
ANSICON=1 R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aaa 2> nul
agC~unk.
echo end
' bins/elf/true
-- stdout
@@ -1 +0,0 @@
-end
-- exit status: -1073741819
[XX] C:\projects\radare2\test\db\cmd\cmd_agC agCd contains label for main
ANSICON=1 R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aaa 2> nul
agCd~label="main"
' bins/elf/true
-- stdout
@@ -1 +0,0 @@
-  "0x00002050" [label="main" URL="main/0x00002050"];
-- exit status: -1073741819
[XX] C:\projects\radare2\test\db\cmd\cmd_agC agCj entry0 imports libc_start_main
ANSICON=1 R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aaa 2> nul
agCj~{0}
' bins/elf/true
-- stdout
@@ -1 +0,0 @@
-{"name":"entry0", "size":46,"imports":["reloc.__libc_start_main"]}
-- exit status: -1073741819
[**]            C:\projects\radare2\test\db\cmd\cmd_afn     3584 OK       108 BR      124 XX        9 FX
[**]           C:\projects\radare2\test\db\cmd\cmd_afcl     3586 OK       114 BR      124 XX        9 FX
[XX] C:\projects\radare2\test\db\cmd\cmd_afb af localflag
ANSICON=1 R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aaa
afi~?
afi.
afb.
' bins/mach0/mac-ls2
-- stdout
@@ -1,3 +0,0 @@
-33
-entry0
-0x1000011e8 0x100001211 01:0899 41 j 0x100001216 f 0x100001211
-- stderr
[ ] Analyze all flags starting with sym. and entry0 (aa)[[x] Analyze all flags starting with sym. and entry0 (aa)
[ ] Analyze function calls (aac)[[x] Analyze function calls (aac)
[ ] Analyze len bytes of instructions for references (aar)[[x] Analyze len bytes of instructions for references (aar)
[ ] Check for objc references[[x] Check for objc references
-- exit status: -1073741819
[**]            C:\projects\radare2\test\db\cmd\cmd_afb     3588 OK       115 BR      125 XX        9 FX
[XX] C:\projects\radare2\test\db\cmd\cmd_agC agCd entry0 imports libc_start_main
ANSICON=1 R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'aaa 2> nul
agCd~2110
' bins/elf/true
-- stdout
@@ -1,2 +0,0 @@
-  "0x00002110" [label=
```
So far changes are sensible, but broken for now.

File: libr/core/anal_objc.c [hunk 1]
```diff
@@ -214,6 +214,7 @@ static bool objc_find_refs(RCore *core) {
 		if (core->anal->verbose) {
 			eprintf ("Could not find necessary Objective-C sections...\n");
 		}
+		core_objc_free (objc);
```
Comment (trufae) [inline libr/core/anal_objc.c:217]: its crasahing because https://github.com/radareorg/radare2/pull/17313 is not merged. ill rebase when this one gets in

**Comment by wargio** [comment]
still red. maybe you need to rebase?

**Comment by trufae** [comment]
it was rebased 18h ago.

**Comment by trufae** [comment]
Just found a regression. lets see if i nailed it this time

File: libr/include/r_core.h [hunk 2]
```diff
@@ -331,6 +331,10 @@ typedef struct r_core_t {
 	RList *ropchain;
 	bool use_tree_sitter_r2cmd;
 
+	/* maybe move this into RCore */
```
Comment (ret2libc) [inline libr/include/r_core.h:334]: i guess the comment is not relevant anymore :D

Comment (wargio) [reply libr/include/r_core.h:334]: i would place that change in a new PR

Comment (ret2libc) [reply libr/include/r_core.h:334]: we can just force merge it if that is the only change actually... we will have to wait anyway for the new PR for the comment :D and doing a PR just for a line of a comment seems a bit... meh!

**Comment by trufae** [comment]
pushed the commit removing the comment

**Pull request merged**